uisrnn/contrib/get_prior_args.py

```diff
@@ -0,0 +1,151 @@
+# Contributor information:
```

Comment:
I've changed module name to range_search_crp_alpha.py

Reply:
> For the file name (module name), I think it's a better idea to make the name more self-explaining - how did you estimate the crp_alpha? I think you are _searching_ for a best parameter value, right? Does it make sense to call it something like `range_search_crp_alpha.py`?
> 
> I'm saying this just in case later someone else wants to contribute another way to estimate `crp_alpha`. We can avoid conflicts here by using more explicit names.
> 
> Also, instead of renaming the module, you can consider moving your functions into a class named `RangeSearchCprAlphaEstimation` as an alternative way.

I've changed module name to range_search_crp_alpha.py

Reply:
I've changd module name to range_search_crp_alpha.py.